samples/CommunityToolkit.Maui.Sample/Platforms/iOS/Info.plist

```diff
@@ -26,6 +26,8 @@
 		<string>UIInterfaceOrientationLandscapeLeft</string>
 		<string>UIInterfaceOrientationLandscapeRight</string>
 	</array>
+	<key>UIViewControllerBasedStatusBarAppearance</key>
+    <false/>
```

Comment:
Just a question - Is this required for users to do Full Screen on iOS / MacCatalyst? 

If so, we'll need to highlight this in our docs!

Reply:
Docs were updated to reflect changes. Look here [https://github.com/MicrosoftDocs/CommunityToolkit/pull/259](https://github.com/MicrosoftDocs/CommunityToolkit/pull/259)